docs/content/getting-started/code-generation.md

```diff
@@ -48,13 +47,13 @@ each has a use case it is best at:
 
 ## Data + Templates
 
-`hof gen interlude.json -T interlude.template` is adhoc template rendering.
+`hof gen interlude.json -T interlude.template` is ad-hoc template rendering.
```

Comment:
should my _adhoc_ be _ad-hoc_ in all the places?

or maybe no hyphen either?

https://www.quora.com/Is-it-adhoc-or-ad-hoc-or-ad-hoc-Which-is-correct

Reply:
sounds good 🔨